src/main/java/org/fogbowcloud/sebal/SEBALHelper.java

```diff
@@ -60,83 +45,78 @@
+
 		int centralMeridian = findCentralMeridian(zoneNumber);
-			
+
 		for (BoundingBoxVertice boundingBoxVertice : boudingVertices) {
 			utmCoordinates.add(convertLatLonToUtm(boundingBoxVertice.getLat(),
-					boundingBoxVertice.getLon(), zoneNumber,
-					centralMeridian));
+					boundingBoxVertice.getLon(), zoneNumber, centralMeridian));
 		}
-		
+
 		LOGGER.debug("Boundingbox UTM coordinates: " + utmCoordinates);
 
 		double x0 = getMinimunX(utmCoordinates);
 		double y0 = getMaximunY(utmCoordinates);
 
 		double x1 = getMaximunX(utmCoordinates);
 		double y1 = getMinimunY(utmCoordinates);
-        
-        double ULx = metadataRoot.getElement("L1_METADATA_FILE")
-                .getElement("PRODUCT_METADATA")
-                .getAttribute("CORNER_UL_PROJECTION_X_PRODUCT").getData()
-                .getElemDouble();
-        double ULy = metadataRoot.getElement("L1_METADATA_FILE")
-                .getElement("PRODUCT_METADATA")
-                .getAttribute("CORNER_UL_PROJECTION_Y_PRODUCT").getData()
-                .getElemDouble();
-        
-        //TODO remove it
-        LOGGER.debug("ULx=" + ULx);
-        LOGGER.debug("ULy=" + ULy);
-        
-        LOGGER.debug("x0=" + x0);
-        LOGGER.debug("y0=" + y0);
-        
-        LOGGER.debug("x1=" + x1);
-        LOGGER.debug("y1=" + y1);
-
-        int offsetX = (int) ((x0 - ULx) / 30);
-        int offsetY = (int) ((ULy - y0) / 30);
-        int w = (int) ((x1 - x0) / 30);
-        int h = (int) ((y0 - y1) / 30);
-
-        BoundingBox boundingBox = new BoundingBox(offsetX, offsetY, w, h);        
-        return boundingBox;
-    }
-    
-	public static int findCentralMeridian(int zoneNumber) throws ClientProtocolException,
-			IOException {
+
+		double ULx = metadataRoot.getElement("L1_METADATA_FILE").getElement("PRODUCT_METADATA")
+				.getAttribute("CORNER_UL_PROJECTION_X_PRODUCT").getData().getElemDouble();
+		double ULy = metadataRoot.getElement("L1_METADATA_FILE").getElement("PRODUCT_METADATA")
+				.getAttribute("CORNER_UL_PROJECTION_Y_PRODUCT").getData().getElemDouble();
+
+		// TODO remove it
+		LOGGER.debug("ULx=" + ULx);
+		LOGGER.debug("ULy=" + ULy);
+
+		LOGGER.debug("x0=" + x0);
+		LOGGER.debug("y0=" + y0);
+
+		LOGGER.debug("x1=" + x1);
+		LOGGER.debug("y1=" + y1);
+
+		int offsetX = (int) ((x0 - ULx) / 30);
+		int offsetY = (int) ((ULy - y0) / 30);
+		int w = (int) ((x1 - x0) / 30);
+		int h = (int) ((y0 - y1) / 30);
+
+		BoundingBox boundingBox = new BoundingBox(offsetX, offsetY, w, h);
+		return boundingBox;
+	}
+
+	public static int findCentralMeridian(int zoneNumber)
+			throws ClientProtocolException, IOException {
 		if (zoneToCentralMeridian.get(zoneNumber) != null) {
 			return zoneToCentralMeridian.get(zoneNumber);
 		} else {
 
 			CloseableHttpClient httpClient = HttpClients.createMinimal();
 
-			HttpGet homeGet = new HttpGet("http://www.spatialreference.org/ref/epsg/327"
-					+ zoneNumber + "/prettywkt/");
+			HttpGet homeGet = new HttpGet(
```

Comment:
extract. it seems, if they replace to another url we need to recompile the code. not good